api.md

```diff
@@ -718,6 +718,9 @@ all attached hypertables.
 | ----------- | -----------            |
 | `node_name` | Name of the data node. |
 
+>:WARNING: Although the `cascade` parameter is not strictly a required argument,
+you *MUST* set it to `true` to avoid breaking necessary user mappings.
```

Comment:
It's more to make sure the user mapping is cleaned up, but I'd probably avoid explaining too much here (maybe just say it must be set to true in the current release).

Reply:
Just removed the user mapping language here!

Reply:
@WireBaron What happens if it is not to set to true? Will it just fail? Or will it fail/succeed with inconsistent state, which we cannot recover?